bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/PanelImpl.java

```diff
@@ -64,4 +69,12 @@ public Type getType() {
         return null;
     }
 
+    @Override
+    public boolean isReadOnly() {
+        if (getEditMode()) {
```

Comment:
Authoring look and feel should be equivalent to that of preview... we only do such things for visible, as that hinders authoring.